CHANGELOG.md

```diff
@@ -1,216 +1,51 @@
-# Release 12.0.0
+# Release 12.1.0
 
 **NOTE:**
-_This changelog only contains release notes for PWA Studio and Venia 12.0.0._
-_For older release notes, see [PWA Studio releases][]._
+_This changelog only contains release notes for PWA Studio and Venia 12.1.0_
+_For older release notes, see_ [PWA Studio releases][].
+
+| Type  | Status       | Assignee                 | Description                                                                           | Jira Issue   | GitHub PR       | Release    |
```

Comment:
Can probably drop the `Status`, `Assignee`, and `Release` columns here.

They'll all be status = released, release = 12.1.
The assignee isn't really relevant to the audience.

Reply:
Yeah, for sure. I added those during dev of the scripts.